### ISSUE TITLE
Windows image consistently failing to build.

### ISSUE THREAD

**Issue by fbattistella** [fluent/fluentd-docker-image#217]
Hi.

I'm building a custom image for windows, which is basically the same one available in the repo (e.g. https://github.com/fluent/fluentd-docker-image/blob/master/v1.10/windows/Dockerfile ) just changing the base image to ```FROM mcr.microsoft.com/windows/servercore:1809```

This used to build just fine last week, but this week it consistently fails on:

```
...
installing mingw-w64-x86_64-tools-git...
Install MSYS2 and MINGW development toolchain succeeded
 _____       _           _____           _        _ _         ___  
|  __ \     | |         |_   _|         | |      | | |       |__ \ 
| |__) |   _| |__  _   _  | |  _ __  ___| |_ __ _| | | ___ _ __ ) |
|  _  / | | | '_ \| | | | | | | '_ \/ __| __/ _` | | |/ _ \ '__/ / 
| | \ \ |_| | |_) | |_| |_| |_| | | \__ \ || (_| | | |  __/ | / /_ 
|_|  \_\__,_|_.__/ \__, |_____|_| |_|___/\__\__,_|_|_|\___|_||____|
                    __/ |           _                              
                   |___/          _|_ _  __   | | o __  _| _     _ 
                                   | (_) |    |^| | | |(_|(_)\^/_> 

Temporarily enhancing PATH for MSYS/MINGW...
Building native extensions. This could take a while...
Successfully installed cool.io-1.5.4
1 gem installed
Temporarily enhancing PATH for MSYS/MINGW...
Building native extensions. This could take a while...
Successfully installed oj-3.3.10
1 gem installed
Temporarily enhancing PATH for MSYS/MINGW...
Building native extensions. This could take a while...
Successfully installed json-2.2.0
1 gem installed
ERROR:  Error installing fluentd:
        ERROR: Failed to build gem native extension.

    current directory: C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6/ext/certstore
C:/ruby26/bin/ruby.exe -I C:/ruby26/lib/ruby/2.6.0 -r ./siteconf20200512-4488-1beltq1.rb extconf.rb
checking for -lcrypt32... yes
checking for PFXExportCertStoreEx() in Wincrypt.h... yes
creating Makefile

current directory: C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6/ext/certstore
make "DESTDIR=" clean

current directory: C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6/ext/certstore
make "DESTDIR="
generating certstore-x64-mingw32.def
compiling certstore.c
compiling certstore_util.c
compiling loader.c
linking shared-object certstore/certstore.so
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x0): multiple definition of `rb_eCertLoaderError'; certstore.o:certstore.c:(.bss+0x0): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x8): multiple definition of `rb_cCertLoader'; certstore.o:certstore.c:(.bss+0x8): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x10): multiple definition of `rb_mCertstore'; certstore.o:certstore.c:(.bss+0x10): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x0): multiple definition of `rb_eCertLoaderError'; certstore.o:certstore.c:(.bss+0x0): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x8): multiple definition of `rb_cCertLoader'; certstore.o:certstore.c:(.bss+0x8): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x10): multiple definition of `rb_mCertstore'; certstore.o:certstore.c:(.bss+0x10): first defined here
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:261: certstore.so] Error 1

make failed, exit code 2

Gem files will remain installed in C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6 for inspection.
Results logged to C:/ruby26/lib/ruby/gems/2.6.0/extensions/x64-mingw32/2.6.0/certstore_c-0.1.6/gem_make.out
```

I tried with the 1.10, 1.9 and 1.7 Dockerfile's in the repo, which in the same windows instance were building just fine last week :/ (which shouldnt matter since.. containers :D).

Hope someone can shed some light on what I can try to get this going, as I dont speak ruby :)

**Comment by repeatedly** [comment]
@cosmo0920 Do you have any insight?

**Comment by cosmo0920** [comment]
This should be fixed in certstore_c v0.1.7.

see: https://github.com/fluent-plugins-nursery/certstore_c/pull/9

**Comment by cosmo0920** [comment]
This should be fixed and I'd uploaded Windows Server Core images:

* https://hub.docker.com/layers/fluent/fluentd/v1.10.3-windows-1.0/images/sha256-527b528cc6c9b32a4799d649cb9ac71ad26cec8c41fa9d3799d2b13aa2476e37?context=explore
* https://hub.docker.com/layers/fluent/fluentd/v1.10.4-windows-1.0/images/sha256-51c2b58017baaf021b13156c857e617b0ef62d5901883cf4a26c27b502ebe6bd?context=explore
